src/main/java/org/apache/sysds/common/Builtins.java

```diff
@@ -38,6 +38,9 @@
  */
 public enum Builtins {
 	//builtin functions
+    CSPLINE("cspline", true),
+	CSPLINE_CG("csplineCG", true),
+	CSPLINE_DS("csplineDS", true),
```

Comment:
we try to keep these in alphabetical order, so input these slightly longer down.